samples/node/pub_sub_electron_node/README.md

```diff
@@ -0,0 +1,108 @@
+
+## Requirements
+
+The sample is built with typescript@5^ and Electron@19. Please note the SDK currently does not support Electron20+.
+Node14 is recommended to run the sample.
+
+**Electron19**
+
+
+## IoT Core Policy
+Your IoT Core Thing's [Policy](https://docs.aws.amazon.com/iot/latest/developerguide/iot-policies.html) must provide privileges for this sample to connect, subscribe, publish, and receive. Below is a sample policy that can be used on your IoT Core Thing that will allow this sample to run as intended.
+
+<details>
+<summary>(see sample policy)</summary>
+<pre>
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iot:Publish",
+        "iot:Receive"
+      ],
+      "Resource": [
+        "arn:aws:iot:<b>region</b>:<b>account</b>:topic/test/topic/*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iot:Subscribe"
+      ],
+      "Resource": [
+        "arn:aws:iot:<b>region</b>:<b>account</b>:topicfilter/test/topic/*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iot:Connect"
+      ],
+      "Resource": [
+        "arn:aws:iot:<b>region</b>:<b>account</b>:client/*"
+      ]
+    }
+  ]
+}
+</pre>
+
+Replace with the following with the data from your AWS account:
+* `<region>`: The AWS IoT Core region where you created your AWS IoT Core thing you wish to use with this sample. For example `us-east-1`.
+* `<account>`: Your AWS IoT Core account ID. This is the set of numbers in the top right next to your AWS account name when using the AWS IoT Core website.
+
+Note that in a real application, you may want to avoid the use of wildcards in your ClientID or use them selectively. Please follow best practices when working with AWS on production applications using the SDK. Also, for the purposes of this sample, please make sure your policy allows a client ID of `test-*` to connect or use `--client_id <client ID here>` to send the client ID your policy supports.
+
+</details>
+
+## How to run
+
+### Direct MQTT via mTLS
+
+To Run this sample using a direct MQTT5 connection with a key and certificate, go to the `node/pub_sub_electron_node` folder.
+1. Setup your credential. You need to fill in the credentials in the `node/pub_sub_electron_node/settings.ts` file with your AWS endpoint, certificate file path, private key file path, and AWS region.
+
+2. Install node packages
+``` sh
+npm install
+```
+
+3. Build and Run
+```sh
+npm run build
+npm run start
+```
+
+## Electron Q&A
+### Warning: `objc[79765]: Class WebSwapCGLLayer is implemented in both ` ?
+
+This is an issue running Electron on MacOS. The API has a name duplication for "WebSwapCGLLayer". The warning should not affect your development. The issue is fixed by Electron in v22. Unfortunately, our SDK currently not supporting Electron@19 and below.
```

Comment:
Nitpick: `... our SDK currently not supporting Electron@19 and below` -> `... our SDK currently only supports Electron@19 and below`.